FireBrick/UI/AlertMessageViewController/SimpleAlertViewController.swift

```diff
@@ -34,7 +34,7 @@ class SingleButtonAlertMessageViewController: UIViewController {
         SwiftEntryKit.dismiss()
     }
     
-    func setAlertView(title: String?, message: String?, buttonTitle: String?) {
+   private func setAlertView(title: String?, message: String?, buttonTitle: String?) {
```

Comment:
remove this method and add 
            headerLabel.text = title
            messageLabel.text = message
            button.setTitle(buttonTitle, for: .normal)
in viewDidLoad